crates/bevy_input/src/button_input.rs

```diff
@@ -116,22 +108,19 @@ use bevy_ecs::schedule::State;
 /// ```
 ///
 /// Accepting input from multiple devices:
-/// ```no_run
+///```no_run
 /// # use bevy_app::{App, NoopPluginGroup as DefaultPlugins, Update};
 /// # use bevy_ecs::{prelude::IntoSystemConfigs, schedule::Condition};
-/// # use bevy_input::{ButtonInput, common_conditions::{input_just_pressed}, prelude::{GamepadButton, Gamepad, GamepadButtonType, KeyCode}};
+/// # use bevy_input::{ButtonInput, common_conditions::{input_just_pressed}, prelude::{ MouseButton, KeyCode}};
 ///
 /// fn main() {
-///     App::new()
+///    App::new()
 ///         .add_plugins(DefaultPlugins)
 ///         .add_systems(
 ///             Update,
 ///             something_used.run_if(
 ///                 input_just_pressed(KeyCode::KeyE)
-///                     .or_else(input_just_pressed(GamepadButton::new(
-///                         Gamepad::new(0),
-///                         GamepadButtonType::West,
-///                     ))),
+///                     .or_else(input_just_pressed(MouseButton::Left)),
```

Comment:
Removing GamepadButton from this example makes it useless in the context of this doc. Given that we're essentially removing the ability to express this in this way for ButtonInput, I think we should just remove the example entirely.